wandb/run_manager.py

```diff
@@ -519,6 +520,10 @@ def mirror_stdout_stderr(self):
         io_wrap.SimpleTee(sys.stderr, streaming_log.TextStreamPusher(
             fs_api, OUTPUT_FNAME, prepend_timestamp=True, line_prepend='ERROR'))
 
+    def unmirror_stdout_stderr(self):
+        sys.stdout.write = sys.stdout.orig_write
+        sys.stderr.write = sys.stderr.orig_write
```

Comment:
Might be better to put this stuff in the SimpleTee class

Reply:
Agreed I’ll do it in a separate pull.